pkg/debuginfo/extract.go

```diff
@@ -86,6 +86,7 @@ func (e *Extractor) Extract(ctx context.Context, dst io.WriteSeeker, src string)
 		isDwarf,
 		isSymbolTable,
 		isGoSymbolTable,
+		isPltSymbolTable,
```

Comment:
Thanks for handling this. I'll review the vdso mapping-related changes later. But just a general comment: as debuginfo-related changes are unrelated to vdso-related changes, let's keep it in the separate PR you've already opened: #1351.

Reply:
open a new pull request https://github.com/parca-dev/parca-agent/pull/1384